packages/theme-default/src/col.css

```diff
@@ -7,20 +7,33 @@
 }
 
 @for $i from 1 to 24 {
-  .el-col-$i,
+  .el-col-$i {
+    width: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-offset-$i {
+    margin-left: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-pull-$i {
+    position: relative;
+    right: calc(1 / 24 * $(i) * 100)%;
+  }
+  .el-col-push-$i {
+    position: relative;
+    left: calc(1 / 24 * $(i) * 100)%;
+  }
+}
+
+@for $i from 1 to 24 {
```

Comment:
@azurelogic Thanks!Your suggestion is very valuable to me. Indeed, i just concerned about the .el-col-xs-XX classes overriding the general .el-col-XX classes. I think i should use the way as you've said. it will be fix in next patch version. However, I still have a question, what's the different between .el-col-XX and .el-col-xs-XX classes?